examples/demo/src/visitors/VisitorCreate.js

```diff
@@ -44,7 +47,11 @@ const VisitorCreate = ({ classes, ...props }) => (
                     fullWidth={true}
                     formClassName={classes.email}
                 />
-                <DateInput source="birthday" />
+                <DateInput source="birthday" formClassName={classes.birthday} />
+                <PasswordInput
```

Comment:
We need to simulate that the clear password will be hashed before it's stored. Unfortunately we don't have API controllers for that...